src/coreclr/jit/gentree.cpp

```diff
@@ -6212,6 +6217,168 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
 #pragma warning(pop)
 #endif
 
+//------------------------------------------------------------------------
+// gtSetEvalOrderMinOpts: A MinOpts specific version of gtSetEvalOrder. We don't
+//    need to set costs, but we're looking for opportunities to swap operands.
+//
+// Arguments:
+//    tree - The tree for which we are setting the evaluation order.
+//
+// Return Value:
+//    the Sethi 'complexity' estimate for this tree (the higher
+//    the number, the higher is the tree's resources requirement)
+//
+unsigned Compiler::gtSetEvalOrderMinOpts(GenTree* tree)
+{
+    assert(tree);
+
+    if (tree->OperIsLeaf())
+    {
+        // Nothing to do for leaves.
+        return 0;
+    }
+
+    unsigned level = 1;
+    if (tree->OperIsSimple())
```

Comment:
nit: Might be worth a small comment saying that we are intentionally not handling hwintrinsics (they otherwise normally support GTF_REVERSE_OPS and other considerations).

Reply:
I'll add a comment, but we skip a lot of trees here, it was driven by asmdiff + tpdiff results

Reply:
> but we skip a lot of trees here

👍, had mostly thought it might be worth a callout because it normally mirrors a lot of the `SimpleOp` handling (unlike any other node), so figured it might be worth a special note/callout